### DISCUSSION TITLE
colors of Draft PR and Closed PR should differ in Dark theme

### DISCUSSION THREAD

**Discussion by hogashi** [community/community#19735] [Pull Requests]
Draft PR and Closed PR are confusing because their colors are both white and icons are similar.  I want to distinguish whether in progress or outdated.  That would be great if the colors are different at least.

| draft | closed |
| :---: | :---: |
| ![image](https://user-images.githubusercontent.com/12870472/176573320-5ae00eba-39ec-475e-afb1-53c100b8ec1d.png) | ![image](https://user-images.githubusercontent.com/12870472/176573350-923324ba-9655-44eb-b305-20dc74b76bd7.png) |

**Comment by thuongtruong1009** [comment]
Hi @hogashi 
I think this is just a small bug in the dark mode interface design. The reason is that you turn on the dark theme for your laptop and Github, so they will lose color. But if you set it to light, you'll see it's dark blue at draft PR and red at close PR.

![image](https://user-images.githubusercontent.com/71834167/176577504-2aa70722-b34d-495a-b397-c2902e30c003.png) and ![image](https://user-images.githubusercontent.com/71834167/176577701-c8207b99-9fe8-4d8e-bfe5-017174be19f3.png)

Hope useful for you!
Have a nice day 🥰

**Comment by hogashi** [comment]
@thuongtruong1009 Oh, indeed I'm using Dark Protanopia & Deuteranopia!  Thanks for your reply.  I hope that bug will be fixed in the dark theme 🌚 

I'm changing this question's title ⚡

**Reply by thuongtruong1009** [reply]
> @thuongtruong1009 Oh, indeed I'm using Dark Protanopia & Deuteranopia! Thanks for your reply. I hope that bug will be fixed in the dark theme 🌚
> 
> I'm changing this question's title ⚡

Oh, great!
If you fix them successfully, can use mark this as the answer for everyone's know about that.
Thank you very much! ❤
Have a nice day!
